desafio1.html

```diff
@@ -21,32 +21,52 @@
 <body>
     <h1>Game Reviews</h1>
     <h2>Crie um novo review</h2>
-    <section>Título do Game</section>
-    <input type="text">
-    <section>Review do Jogo</section>
-    <textarea></textarea> 
-    <form>
-        Categoria do Jogo
-            <select name="categoriajogo">
+    <form id="new_review_form" >
+    <label for="title">Título do Game</label>
+    <input id="title" type="text">
+    <br>
+    <label for="review">Review do Jogo</label>
+    <textarea id="review"></textarea> 
+    <br>
+    <label for="category">Categoria do Jogo</label>
+            <select id="category" name="categoriajogo">
             <option value="">Escolha a categoria</option>
             <option value="FPS">FPS</option>
             <option value="Indie">Indie</option>
             <option value="Estratégia">Estratégia</option>
             <option value="Lógica">Lógica</option>
         </select>
-           </form>
+        
     <br>         
-    <section>Nota do Review</section>
-    <input type="radio">
-    <input type="radio">
-    <input type="radio">
-    <input type="radio">
-    <input type="radio">
+    <label>Nota do Review</label>
+    <input name="rating" type="radio" value="1">
+    <input name="rating" type="radio" value="2">
+    <input name="rating" type="radio" value="3">
+    <input name="rating" type="radio" value="4">
+    <input name="rating" type="radio" value="5">
     <br>
     <button class="btn" type="submit">Salvar Review</button>
+    </form>
     <script>
-        var salvar = document.querySelector
+        // 
+        const form = document.querySelector("#new_review_form");
```

Comment:
Estou recuperando o elemento do formulário usando o id que foi atribuído ao elemento <form>
Fica mais fácil de recuperar elementos usando seus ID's únicos.

Note que eu adicionei um jogo da velha (**`#`**) ao início da string. Isso indica para o Javascript que vc está buscando pelo atributo **ID** de um elemento.